workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -168,20 +168,9 @@ rule deploy:
     benchmark:
         "benchmarks/deploy.txt"
     conda: config["conda_environment"]
-    shell:
-        """
-        nextstrain deploy {params.deploy_url:q} {input:q}
-
-        if [[ -n "$SLACK_TOKEN" && -n "$SLACK_CHANNEL" ]]; then
-            curl https://slack.com/api/chat.postMessage \
-                --header "Authorization: Bearer $SLACK_TOKEN" \
-                --form-string channel="$SLACK_CHANNEL" \
-                --form-string text={params.slack_message:q} \
-                --fail --silent --show-error \
-                --include
-        fi
-        """
-
+    run:
+        shell("nextstrain deploy {params.deploy_url:q} {input:q}")
+        send_slack_message(message)

```

Comment:
This needs to access the snakemake param `params.slack_message`

Reply:
Thanks!  4675062